lms/product/plugin/misc.py

```diff
@@ -96,7 +97,12 @@ def get_deeplinking_launch_url(self, request, _assignment_configuration: dict):
     def get_deep_linked_assignment_configuration(self, request) -> dict:
         """Get the configuration of an assignment present in the current launch deep link."""
         params = {}
-        possible_parameters = ["url", "group_set", "deep_linking_uuid"]
+        possible_parameters = [
+            "url",
+            "group_set",
+            "deep_linking_uuid",
+            "auto_grading_config",
```

Comment:
This method returns the raw values from the deep linked configuration but only a known subset of them. Include auto grading now.